Styles.css

```diff
@@ -0,0 +1,342 @@
+  align-items: center;
+}
+
+.Introduction__p {
+  text-align: center;
+  display: block;
+  font-weight: 200;
+  margin-top: 1.25em;
+  font-size: 1.15em;
+}
+
+.Introduction__avatar {
+  margin-top: 2em;
+  width: 170px;
+  display: inline-block;
+}
+
+.Introduction__heroMachine {
+  width: 520px;
+  margin-top: 3.7em;
+}
+
+.personal-info-container {
+  background-color: #7510f7;
+  display: flex;
+  height: 500px;
+  align-items: center;
+  flex-direction: column;
+  position: relative;
+}
+
+.personal-info-container__header {
+  text-align: center;
+  margin-top: 100px;
+}
+
+h2 {
+  color: white;
+  font-family: "Eurostile Extended", sans-serif;
+  font-size: 1.3em;
+}
+
+.personal-info-container__introduction {
+  padding-top: 1.5em;
+  color: white;
+  width: 620px;
+  text-align: center;
+  display: inline-block;
+  line-height: 1.7em;
+  font-weight: 100;
+  font-family: "Roboto", sans-serif;
+}
+
+.card-info {
+  display: grid;
+  grid-template-columns: repeat(1, 400px);
+  justify-content: center;
+  /* margin-top: -8em; */
+  position: absolute;
+  /* left: 50%;
+  top: -3em; */
+  top: 25em;
+}
+
+.card-info__skills {
+  background-color: white;
+  width: 400px;
+  height: 600px;
+  border: 1px solid #e6ecf8;
+  border-radius: 24px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+ .card-info__skills__svg {
+  height: 50px;
+  width: 48px;
+  padding-top: 3.125em;
+}
+
+.card-info__skills h2 {
+  text-align: center;
+  display: inline-block;
+  width: 250px;
+  font-size: 1.5em;
+  color: black;
+  margin-top: 2rem;
+}
+
+.card-info__skills__personal {
+  width: 310px;
+  font-weight: 100;
+  color: black;
+  display: inline-block;
+  text-align: center;
+  margin-top: 2em;
+}
+
+
+.card-info__skills h3,
+.card-info__skills h4 {
+  color: #7510f7;
+  font-weight: 200;
+  margin-top: 1.875em;
+  text-align: center;
+  font-size: 1.3125rem;
+}
+
+.card-info__skills__lang {
+  width: 310px;
+  color: black;
+  margin-top: 1.875em;
+  display: inline-block;
+  text-align: center;
+  font-weight: 300;
+}
+
+.card-info__skills__list {
+  margin-top: 1.5em;
+  text-align: center;
+}
+
+.card-info__skills__list__item {
+  list-style: none;
+  font-weight: 200;
+  margin-bottom: 0.5em;
+}
+
+.recent-work-container {
+  /* margin-top: 8.125em; */
+  margin-top: 37.125em;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.recent-work-container__h2 {
+  color: black;
+  text-align: center;
+  font-size: 1.5em;
+}
+
+.recent-work-container__p {
+  margin-top: 1.5em;
+  font-weight: 300;
+}
+
+.project-grid {
+  display: grid;
+  grid-template-columns: repeat(3, 300px);
+  justify-content: center;
+  gap: 20px;
+  margin-top: 4em;
+}
+
+.project-grid__item {
+  position: relative;
+}
+
+.project-grid__img {
+  height: 200px;
+  width: 300px;
+  border-radius: 12px;
+  cursor: pointer;
+}
+
+.project-overlay {
+  color: white;
+  position: absolute;
```

Comment:
same here

Reply:
I need it here in order for the hover to work